edweiss-app/components/core/containers/TView.tsx

```diff
@@ -1,14 +1,15 @@
 import ReactComponent from '@/constants/Component';
 
 import { ContainerProps, computeContainerStyle } from '@/constants/Style';
-import { useOptionalColor, useOptionalThemeColor } from '@/hooks/theme/useThemeColor';
 import { View, type ViewProps } from 'react-native';
 
 export type TViewProps = ViewProps & ContainerProps;
 
 const TView: ReactComponent<TViewProps> = ({ backgroundColor, borderColor, style, columnGap = 0, rowGap = 0, ...props }) => {
-	const computedBackgroundColor = useOptionalThemeColor({ light: props.light, dark: props.dark }, backgroundColor);
-	const computedBorderColor = useOptionalColor(borderColor);
+	//const computedBackgroundColor = useOptionalThemeColor({ light: props.light, dark: props.dark }, backgroundColor);
```

Comment:
You can delete useless comments

Reply:
This is the work of @TuanDangNguyen2003, he found a bug occuring while running the test. We will fix it with @holoisme as he has a brawther view on it core functionality.